.github/workflows/sanity.yml

```diff
@@ -11,16 +11,14 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         version: [8, 11, 17]
-        impl: [hotspot]
+        distribution: [temurin]
         buildlist: [openjdk, system, functional]
     steps:
     - uses: actions/checkout@v3
-    - uses: AdoptOpenJDK/install-jdk@v1
+    - uses: actions/setup-java@v3
       with:
-        version: ${{ matrix.version }}
-        targets: JDK_${{ matrix.version }}
-        impl: ${{ matrix.impl }}
-        source: 'nightly'
+        java-version: ${{ matrix.version }}
+        distribution: ${{ matrix.impl }}
```

Comment:
`distribution` is required by setup-java action. In your change line 14 if you change the impl parameter to be `distribution` ${{ matrix.distribution }} should be used.

Reply:
:facepalm: Sorry for the typo.